pkg/provisioners/multichannelfanout/config.go

```diff
@@ -18,7 +18,7 @@ package multichannelfanout
 
```

Comment:
Should this be nested inside `in-memory`? Do we expect any other provisioners to use it?

Reply:
kafka uses the config, hence I'll leave it under provisioners.